lld/test/ELF/debug-names-bad-die-idx-sizes.s

```diff
@@ -1,10 +1,10 @@
-# This file was generated by first compiling main.cpp:
-# clang++ -g -O0 -S -gpubnames -fdebug-compilation-dir='debug-names-test' \
-#      main.cpp
+# This file was generated by first compiling a.cpp:
+# clang++ -g -O0 -S -gpubnames -fdebug-compilation-dir='/proc/self/cwd' a.cpp
+
 # Then manually edit the .debug_names section. In the entries, change the
 # size of the DW_IDX_die_offset from '.long' to '.byte'.
 
-# Contents of main.cpp:
+# Contents of a.cpp:
 # int main (int argc, char **argv) { }
```

Comment:
Probably simpler DWARF without the parameters (`int main() { }`) - or, even better (if it gets far enough through lld, with `void f1() { }`, but lld might bail out too early for that)

Reply:
Do you feel strongly about this? I'd really prefer not to regenerate all the .s files again...

Reply:
I don't feel strongly about this. But if we use the sed pattern (
https://github.com/llvm/llvm-project/pull/86508#issuecomment-2050247449) for debug-names-bad-name-count.s debug-names-bad-die-idx-sizes.s debug-names-invalid-parent-idx.s and possibly others, we can potentially remove future test maintenance burden.

Reply:
I might get around to cleaning this up in a follow-up (out of curiosity, I found the difference between `int main() { }` built with `-g -gpubnames` and `int main(int argc, char** argv) { }` is 279 lines for the latter, 211 lines for the former - figured it might be worse, but still a substantial reduction) - yeah, perhaps revisiting the tests with the sed pattern @MaskRay has pointed to would be an opportunity to reduce the duplication further, and only have to regenerate one, or at least fewer, tests.